packages/esm-appointments-app/src/form/appointments-form.component.tsx

```diff
@@ -128,6 +131,10 @@ const AppointmentsForm: React.FC<AppointmentsFormProps> = ({
     resolver: zodResolver(appointmentsFormSchema),
     defaultValues: {
       location: appointment?.location?.uuid ?? session?.sessionLocation?.uuid ?? '',
+      provider:
+        appointment?.providers?.find((provider) => provider.response === 'ACCEPTED')?.uuid ??
+        session?.currentProvider.uuid ??
+        '', // assumes only a single previously-scheduled provider with state "ACCEPTED", if multiple, just takes the first
```

Comment:
So it looks like the appointments API the "provider" reference on appointment has been replaced by a patient_appointments_provider table to allow multiple providers (though, tangent, the provider column has not been removed though it doesn't seem to be used anymore). When the provider is changed, the old provider is marked as "CANCELLED". The current provider is the one marked as "ACCEPTED". I believe that this (repopulating the provider on edit) didn't work with the old code.